libs/src/services/creatorNode/index.js

```diff
@@ -19,6 +19,19 @@ class CreatorNode {
    */
   static getSecondaries (endpoints) { return endpoints ? endpoints.split(',').slice(1) : [] }
 
+  /**
+   * Pulls off the user's clock value from a creator node endpoint and the user's wallet address.
+   * @param {string} endpoint creator node endpoint
+   * @param {string} wallet user wallet address
+   */
+  static async getClockValue (endpoint, wallet) {
```

Comment:
ignore this, duped from https://github.com/AudiusProject/audius-protocol/pull/1121/files